scripts/cog.sh

```diff
@@ -0,0 +1,8 @@
+
+for file in `ls *.tif *.tiff`; do
```

Comment:
What does this do?

Reply:
Added comments - this script takes geotiffs and makes them cloud optimized.